### PR TITLE
correcting swapped line sample prints in enlarge application output file

### PR DESCRIPTION
**Pull request by ladoramkershner**
Corrected the enlarge application output file prints.

## Description

Swapped the line and sample variables in the enlarge output file prints to accurately represent values.

## Related Issue

Issue #3659 

## Motivation and Context

Issue #3659 

## How Has This Been Tested?

Existing tests for enlarge were run and passed. Since this is just an output file change and does not change the functionality of the application.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read and agree to abide by the [Code of Conduct](https://usgs-astrogeology.github.io/code/)
- [X] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [X] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Review by krlberry — DISMISSED** [review]
:+1:

**Review by jessemapel — COMMENTED** [review]
Change is good, but this probably slipped through because the results group generated by this is never checked in the unittest! A quick fix would be to update the old style unit test to capture the output of this function and print it, but it's probably a good candidate for converting to gtest. Things get a little complicated with the SubArea class, but it all looks do-able via mocks.

**Comment by ladoramkershner** [comment]
Is there a place where we can make a note of this change for the gtest conversion? I see that there is a trello card for the conversion of the enlarge test, but this seems like a larger issue and we may want to test all application outputs. I am not sure where to put that information/request.

**Comment by jessemapel** [comment]
@ladoramkershner We could open a new issue and fix this as a separate PR. I think it makes more sense to make the fix to the tests under this PR though.

**Comment by ladoramkershner** [comment]
But don't we want it on every application that has an output? Making one issue for all application with a write out seems like too large of an issue, but making one issue per application sounds tedious. I think making a note that it should be added in during gtest conversion would be good (although very similar to making one large blanket issue)

**Comment by jessemapel** [comment]
> But don't we want it on every application that has an output?

I am unsure of what you are reference with it. I'm concerned that the output of this function is being discarded in the unit test:

https://github.com/USGS-Astrogeology/ISIS3/blob/dev/isis/src/base/objs/Enlarge/unitTest.cpp#L34

**Comment by ladoramkershner** [comment]
I am meaning to say, is it common practice to test the output file of a function in isis unit tests? If yes, then a single ticket makes sense. If not, then one large ticket to say we should START testing file outputs seems non-digestable in any single sprint.

**Comment by jessemapel** [comment]
Ah. I'm not talking about file output, but the actual return of the function. Enlarge::UpdateOutputLabel returns a PvlGroup and in the unit test for the Enlarge class, it is simply thrown away and never checked. The unit test should be updated to properly check the function return so that this doesn't regress.

**Comment by ladoramkershner** [comment]
Oh okay. Sorry! Okay I can make a ticket for this!

**Comment by ladoramkershner** [comment]
So I added the Results group into the truth file, but I am realizing that the isisTruth.cub used for the test is square, so it would not have caught the error Aaron reported anyways 😆

File: isis/src/base/objs/Enlarge/Enlarge.truth [hunk 1]
```diff
@@ -1,4 +1,16 @@
...
+  InputSamples    = 126
+  StartingLine    = 1
+  StartingSample  = 1
+  EndingLine      = 126
+  EndingSample    = 126
+  LineIncrement   = 0.5
+  SampleIncrement = 0.5
+  OutputLines     = 252
+  OutputSamples   = 252
+End_Group
```
Comment (jessemapel) [inline isis/src/base/objs/Enlarge/Enlarge.truth:13]: Make sure that you update unittest.cpp to output this too!

**Review by jessemapel — APPROVED** [review]
Looks good. Once CI passes this can be merged.

**Pull request merged**